Methods/Parameters/Inventory/Inventory/Program.cs

```diff
@@ -0,0 +1,314 @@
+        Console.WriteLine($@"Item successfully removed.");
+    }
+    else
+    {
+        Console.WriteLine("Item was not removed.");
+    }
+
+    ReturnToMenu();
+}
+string FindItem()
+{
+    string findname ="";
+    bool itemnotfound = true;
+    while (itemnotfound)
+    {
+
+        findname = Console.ReadLine();
+        while (string.IsNullOrWhiteSpace(findname))
+        {
+            Console.WriteLine("Invalid input. Please enter a valid name.");
+            findname = Console.ReadLine().ToUpper();
+        }
+        foreach (Item i in items)
+        {
+            if (i.Name == findname)
+            {
+                itemnotfound = false;
+                
+            }
+
+
+        }
+
+        if (itemnotfound) { 
+        Console.Clear();
+        Console.WriteLine("Name not found. Try again.");
+        }
+    }
+    return findname; ;
+}
+void AddItemDisplay()
+{ //static (name, int, int) AddItemDisplay(){}
+    Console.WriteLine(@"
+        ----------------------------Add New Item ----------------------------
+    Please enter the item name:");
+    string name = Console.ReadLine();
+    while (string.IsNullOrWhiteSpace(name))
+    {
+        Console.WriteLine("Invalid input. Please enter a valid name.");
+        name = Console.ReadLine().ToUpper();
+    }
+    Console.WriteLine(@"
+    Please enter the item quantity:");
+    int quantity;
+    while (!int.TryParse(Console.ReadLine(), out quantity) || quantity > 100)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for quantity:");
+    }
+    Console.WriteLine(@"
+    Please enter the item price per unit:");
+    int price;
+    while (!int.TryParse(Console.ReadLine(), out price) || price > 100)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for price:");
+    }
+    Additem(name, quantity, price);
+}
+void Additem(string name, int quantity, int price) {
+    
+    items.Add(new Item { Name = name, Quantity = quantity, Price = price });
+    Console.WriteLine(@"
+    --------------------------------------------------------------------");
+
+    Console.WriteLine($@"Item successfully added to the inventory.");
+
+    ReturnToMenu();
+}
+void ViewInventory()
+{
+    Console.WriteLine(@$"
+    ----------------------------Inventory----------------------------
+    | Item Name       | Quantity | Price(per unit)| Total Value |
+    ---------------------------------------------------------------");
+
+   
+
+    foreach (Item i in items)
+    {
+        Console.WriteLine(@$"    | {i.Name,-15} | {i.Quantity,8} | {i.Price,14} | {i.Quantity * i.Price,11} |");
+        
+    }
+    CalculateTotal();
+
+    Console.WriteLine($@"    ---------------------------------------------------------------
+    Total Inventory Value: {totalvalue};
+    -----------------------------------------------------------------");
+
+    ReturnToMenu();
+
+}
+static void MainMenu()
+{
+    Console.WriteLine(@"
```

Comment:
The main menu text is hardcoded in MainMenu. Consider using a string constant or a method that dynamically generates the menu.

ex:

`const string MainMenuText = @"
---------------------------- Simple Inventory System ----------------------------
1. View Inventory
2. Add Item to Inventory
3. Remove Item from Inventory
4. Update Item in Inventory
5. Search Item by Name
6. View Total Inventory Value
7. Exit
---------------------------------------------------------------------------------
Please select an option (1-7):";
Console.WriteLine(MainMenuText);
`